wordpress.org/public_html/wp-content/plugins/photo-directory/inc/uploads.php

```
@@ -480,19 +480,27 @@ public static function customize_fu_response_notices( $notices ) {
+		$fields = [
+			'post_title'   => 'sanitize_text_field',
+			'post_content' => 'sanitize_textarea_field',
+			'post_excerpt' => 'sanitize_text_field',
+		];
+
+		foreach ( $fields as $field => $sanitize ) {
+			if ( ! isset( $post_array[ $field ] ) ) {
+				continue;
+			}
+
+			$value = $sanitize( wp_unslash( $post_array[ $field ] ) );
```

PHPStan (2.2.8)

[error] Trying to invoke 'sanitize_text_field'|'sanitize_textarea…' but it's not a callable.


(callable.nonCallable)